src/themes/default/elements/icon.overrides

```diff
@@ -174,6 +174,7 @@ i.icon.help:before { content: "\f128"; }
 i.icon.info:before { content: "\f129"; }
 i.icon.warning:before { content: "\f12a"; }
 i.icon.birthday:before {  content: "\f1fd"; }
+i.icon.help.circle.outline:before { content: "\f29c"; }
```

Comment:
New icon `help circle outline`, the "?" symbol with an outlined circle. Not to be confused with `help circle`, the "?" symbol, with a circular solid background.